src/views/components/EntityCard.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+	<div class="entity-card" :style="getCardStyle()">
+		<img class="img-tier" 
+			:class="{'user-tier': !this.isOpponent(), 'opponent-tier': this.isOpponent()}" 
+			:src="getEntityTierPath()" />
+
+		<img class="img-entity" 
+			:class="{'user-pet': !this.isOpponent(), 'opponent-pet': this.isOpponent()}" 
+			:src="getEntityPath()" />
+
+		<img class="img-pedestal" src="../../assets/images//layout/shop-pedestal-with-shadow.png" />
+
+		<template v-if="isPet()">
+			<img class="img-stats" src="../../assets/images/layout/animal-stats-template.png" />
+			<span class="card-text text-attack">{{getPetAttack()}}</span>
```

Comment:
when referencing methods from moustache template you used to be able to leave out the () on the method name, might have changed from vue2 to vue3 but that's how it used to work i think?